src/dashboard/index.js

```diff
@@ -37,6 +38,230 @@ function render() {
         }
       });
   });
+
+    document.getElementById("activity").addEventListener("click", () => {
+        const repository = path.join("file://", __dirname, "activity.html");
+        let os = require("os");
+        fetch(repository)
+            .then((response) => response.text())
+            .then((text) => {
+                document.querySelector("#content").innerHTML = text;
+                var cookbook_activity = document.createElement('div');
```

Comment:
any specific reason why we use `var` instead of `const`

Reply:
@sanjain-progress  const object properties cannot be changed, but i am changing it by setting attributes and updating them by appending the child object.